buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -454,6 +455,7 @@ class BeamModulePlugin implements Plugin<Project> {
         google_api_services_clouddebugger           : "com.google.apis:google-api-services-clouddebugger:v2-rev20191003-$google_clients_version",
         google_api_services_cloudresourcemanager    : "com.google.apis:google-api-services-cloudresourcemanager:v1-rev20191206-$google_clients_version",
         google_api_services_dataflow                : "com.google.apis:google-api-services-dataflow:v1b3-rev20190927-$google_clients_version",
+        google_api_services_healthcare              : "com.google.apis:google-api-services-healthcare:v1alpha2-rev20190901-$google_cloud_healthcare_version",
```

Comment:
Good question!
Please let me know if I'm misunderstanding but I believe that is [set to 1.30.3](https://github.com/apache/beam/blob/master/buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy#L379)
And looking at this [repo for healthcare client](https://github.com/googleapis/google-api-java-client-services/tree/master/clients/google-api-services-healthcare/v1beta1) there doesn't seem to be anything past 1.30.1 (which was recently regenerated).

Also FYI you're commenting on an outdated commit we are now using beta client lib.

Reply:
However the generated README has version listed as v1beta1-rev20200327-1.30.9
so maybe I am misunderstanding.

Reply:
It'd make this PR cleaner to use this most recent version because [this commit adds `SchematizedData`](https://github.com/googleapis/google-api-java-client-services/commit/48db535eb30fea17c615f9951d7afad6f3ea72b4#diff-89048b80fe6bcee9b0aade84024aea66).

which I am currently around [here](https://github.com/apache/beam/pull/11151/files#diff-2173a334dc15f8b7dc3257efeec64504R27).

Reply:
@lukecwik Is there a reason this needs to be `google_clients_version` / 1.30.3?
I've just tried testing with that version and it breaks most usage of HCAPI.

Reply:
It doesn't have to be 1.30.3, it could be any later version. The purpose of them being the same is to ensure that GCP dependencies are compatible with each other so the healthcare dep doesn't break other GCP deps and vice versa.

Reply:
I'm rather new to these generated clients and beam.
Is there any process this community has followed in the past for finding a suitable version in situations like this?
Seems like a lot for this PR to make this rather global upgrade.
Naturally if I just flip google_clients_version to 1.30.9 it breaks  a lot:
```
Could not determine the dependencies of task ':sdks:java:harness:shadowJar'.
> Could not resolve all dependencies for configuration ':sdks:java:harness:runtimeClasspath'.
   > Could not find com.google.apis:google-api-services-cloudresourcemanager:v1-rev20191206-1.30.9.
     Required by:
         project :sdks:java:harness > project :sdks:java:extensions:google-cloud-platform-core
   > Could not find com.google.apis:google-api-services-storage:v1-rev20191011-1.30.9.
     Required by:
         project :sdks:java:harness > project :sdks:java:extensions:google-cloud-platform-core
   > Could not find com.google.oauth-client:google-oauth-client:1.30.9.
   > Could not find com.google.apis:google-api-services-storage:v1-rev20191011-1.30.9.
     Required by:
         project :sdks:java:harness > project :sdks:java:extensions:google-cloud-platform-core > com.google.cloud.bigdataoss:gcsio:2.0.0
         project :sdks:java:harness > project :sdks:java:extensions:google-cloud-platform-core > com.google.cloud.bigdataoss:util:2.0.0
   > Could not find com.google.oauth-client:google-oauth-client:1.30.9.
     Required by:
         project :sdks:java:harness > project :sdks:java:extensions:google-cloud-platform-core > com.google.cloud.bigdataoss:gcsio:2.0.0
         project :sdks:java:harness > project :sdks:java:extensions:google-cloud-platform-core > com.google.cloud.bigdataoss:util:2.0.0
   > Could not find com.google.oauth-client:google-oauth-client-java6:1.30.9.
     Required by:
         project :sdks:java:harness > project :sdks:java:extensions:google-cloud-platform-core > com.google.cloud.bigdataoss:gcsio:2.0.0
         project :sdks:java:harness > project :sdks:java:extensions:google-cloud-platform-core > com.google.cloud.bigdataoss:util:2.0.0
   > Could not find com.google.oauth-client:google-oauth-client-java6:1.30.9.
     Required by:
         project :sdks:java:harness > project :sdks:java:extensions:google-cloud-platform-core > com.google.cloud.bigdataoss:gcsio:2.0.0 > com.google.api-client:google-api-client-java6:1.30.9
* 
```

seems like our culprits are 
- [ ] `google-oauth-client`
- [ ] `google-oauth-client-java6`
- [ ] `google-api-services-storage`

If I go in and try to massage the revision numbers to make this work how will I ensure that all those reliant on those deps are still working? Is that covered in pre/post commit ITs?

Reply:
https://cwiki.apache.org/confluence/display/BEAM/Dependency+Upgrades

Combining the linkage checker with a bunch of precommits and postcommits gives us the confidence that the change is safe.

The linkage checker is a tool which finds linkage errors by scanning the classpath and finding missing classes/methods. You run the linkage checker on all modules before and after your change and show that the linkage errors reduced or stayed the same. Any new linkage errors need to be followed up and explained why they are safe.

Reply:
Typically knowing which versions you can try is very hard without a lot of experience with a specific set of dependencies (e.g. knowing a lot about jackson echo system vs gRPC vs ...).

You should find a google-cloud-bom and matching libraries-bom that contain the healthcare dep you want to add. That should give you hints as to what are all the versions of things that work together. You will still need to perform the linkage checker analysis and run the tests as described above.

Reply:
It turns out that someone did the work to upgrade google_clients_version to 1.30.9 in #11208